src/scenes/home/getInvolved/successStories/successStoriesContent.json

```diff
@@ -0,0 +1,22 @@
+{
+  "items": [
```

Comment:
YAS 💃 

I love content file. Instead of `.json`, just do a `.js` file and then you can dynamically import images [like so](https://github.com/OperationCode/operationcode_frontend/blob/master/src/scenes/home/team/boardMembers.js)